nixos/download.tt

```diff
@@ -135,44 +135,74 @@ of NixOS  using azure-cli, just run:</p>
 <pre>
 # settings
 # vm name must be lowercase and letters/numbers only because it's also used as the domain name
-$ export VM=nixvm005
-$ export GROUP=test-group
-$ export LOCATION=westus
-$ export STORAGE=testst004
-$ export SOURCE_URI=<tt>[% blob %]</tt>
+export VM_NAME="nixosvm"
+export RESOURCE_GROUP="rg-nixosvm"
+export LOCATION="eastus"
+export STORAGE="nixosvm"
+export SOURCE_URI="<tt>[% blob %]</tt>"
 
 # create group
-$ azure config mode arm
-$ azure group create $GROUP --location $LOCATION
+az group create \
+    --name "$RESOURCE_GROUP" \
+    --location "$LOCATION
 
 # create storage
-$ azure storage account create $STORAGE -g $GROUP -l $LOCATION --type LRS
-$ azure storage account keys list $STORAGE -g $GROUP
+az storage account create \
+    --name "$STORAGE_ACCOUNT" \
```

Comment:
It looks like STORAGE_ACCOUNT is unset prior to here

Reply:
Resolved and updated.